### ISSUE TITLE
Outline Lighting issue in a few worlds

### ISSUE THREAD

**Issue by Alucard0504** [MochiesCode/Mochies-Unity-Shaders#3]
This is mainly due to poor lighting in worlds. but there is some instances where the model will appear pitch black and even end up over rendering the menu in some worlds when the lighting fails. This is mainly on the outline shader. If you dont use outline it seems to work fine.

Update: The one thats working is using v1.1, the one thats not is using v1.2. And gun on back is using normal shader. so if it was outline that was broken the gun should still be visible. so v1.2 is broken. has some weird issues in worlds with poor lighting

![unknown](https://user-images.githubusercontent.com/67936173/111051020-c5a47080-8415-11eb-8b5c-2c09ef311722.png)
![unknown (3)](https://user-images.githubusercontent.com/67936173/111051021-c63d0700-8415-11eb-9bfd-a6af195197ed.png)

**Comment by tsumeone** [comment]
Not sure if this is related to the outline lighting issue, but I also noticed a strange lighting problem on 1.12.  After updating from 1.10 to 1.12 I noticed my eyes got really dark in worlds with poor lighting.  In my case it seemed to be the option "Lighing-based IOR" under reflections.  I went back to 1.10 since I like the effect Lighting-based IOR gives.  The issue was really noticeable in the MMD world.


1.10:
![1 10](https://user-images.githubusercontent.com/63486880/111182281-24d4c300-8585-11eb-8fe5-db71f8a7e90c.jpg)

1.12:
![1 12](https://user-images.githubusercontent.com/63486880/111182290-27371d00-8585-11eb-8ebb-74d08c3e5d9c.jpg)
